app/views/main.scala.html

```diff
@@ -19,6 +19,7 @@
 	<link rel="apple-touch-icon" sizes="114x114" href="@assets("images/favicons/114x114.png")">
 	<link rel="apple-touch-icon" sizes="72x72" href="@assets("images/favicons/72x72.png")">
 	<link rel="apple-touch-icon-precomposed" href="@assets("images/favicons/57x57.png")">
+	<script type="text/javascript" async src="@assets("googleTagManager.js")"></script>
```

Comment:
Will this script be render blocking? Probably it is worth to double check if google tag manager will work correctly by putting `defer` in the script tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script

Reply:
None of it is blocking, it will work with defer but as I understand it there is no real difference between async which is there and defer (or at least it depends on the browser). ls there a reason you think defer is better?

Reply:
It is fine, thanks for the clarification!